inc/api.php

```diff
@@ -333,7 +333,7 @@ function expectApplied($filter)
      *
      * @param string $filter
      * @param null   $callback
-     * @return bool
+     * @return false|int Return hook priority or false if hook was not added.
```

Comment:
I'm not sure this is right.

If I do `has_filter( 'the_content', 'my_filter' )`, I will either get `false`, if the filter callback has not been added, or the priority, if it did.

However, if I **don't* pass a callback, and just ask for something like `has_filter( 'the_content' )`, I would expect to get either `true` or `false`...?

Reply:
I also was confused about this. But this function works as counter part of Worpdress `has filter`, isn't it?
https://core.trac.wordpress.org/browser/tags/5.6/src/wp-includes/plugin.php#L132

Reply:
I rechecked code: yes, can return `true`:
https://github.com/Brain-WP/BrainMonkey/blob/master/src/Hook/HookStorage.php#L217

Reply:
Indeed the proper doc should be `bool|int` because the function _is_ the counter part of WordPress `has_filter`, and even if WordPress `has_filter` is documented as `false|int` it actually returns `true` when no callback is given and there's at least a callback attached.

In fact, WordPress `has_filter` calls `WP_Hook::has_filter` which, when no callback is given, calls [`WP_Hook::has_filters`](https://github.com/WordPress/WordPress/blob/master/wp-includes/class-wp-hook.php#L203), which [will return `true` if any callback is there](https://github.com/WordPress/WordPress/blob/9b44c28f786d0321bb7f632ca9271ca75a5f07fe/wp-includes/class-wp-hook.php#L230).

So @shvlv would you mind to change the doc-block to `bool|int` so I can merge?

Thanks @tfrommen for looking into it.

Reply:
@gmazzap Done.

Reply:
Thanks @shvlv